### ISSUE TITLE
`UnusedUnaryOperator` false positive with `var` property setter and `if` expression

### ISSUE THREAD

**Issue by snyman** [detekt/detekt#5093]
## Expected Behavior

```
class Box {
	var x: Int = 0
}

fun setX(box: Box, shouldBeNegative: Boolean) {
	box.x = if (shouldBeNegative) {
		-1
	} else {
		1
	}
}
```
Should not trigger the `UnusedUnaryOperator` rule.

## Observed Behavior

The rule is triggered:
```
This 'box.x = if (shouldBeNegative) {
                -1
        } else {
                1
        }' is not used [UnusedUnaryOperator]
```

## Steps to Reproduce

See code snippet.

## Context

The snippet is fairly similar to the actual code. I am trying to set a `var` property with an `if` expression where one of the branches is just a constant negative value. I can work around it by bringing the negative constant onto the same line as the if (e.g. `box.x = if (shouldBeNegative) -1 else 1`), however this doesn't seem to be the intent of the rule, nor would it work in all situations, depending on the length of the condition and the codestyle being used in the project.

**Comment by t-kameyama** [comment]
I will work on this issue.
